editor/debugger/editor_profiler.cpp

```diff
@@ -423,6 +424,15 @@ void EditorProfiler::_notification(int p_what) {
 		case NOTIFICATION_TRANSLATION_CHANGED: {
```

Comment:
I'm curious why `NOTIFICATION_TRANSLATION_CHANGED` is required here in the first place.

Reply:
Yeah, I was unsure why that was there in the first place, I could take it out but it's not really doing much harm at any rate